### PR TITLE
changelog-november-01

### PR DESCRIPTION
**Pull request by d-stefanovic**

### CODE REVIEW

**Comment by csidyel** [comment]
What's wrong with the check?

**Comment by d-stefanovic** [comment]
gcloud credentials were changed

**Review by csidyel — APPROVED** [review]

File: docs/reference/semaphore-changelog.md [hunk 1]
```diff
@@ -8,6 +8,37 @@ Thank you for using Semaphore!
...
+
+- Erlang 22.3.4.23
+
+**(Updated)** Updated packages:
+
+- Azure-cli 2.29.2 -> 2.30.0
+- Aws-cli 1.21.7 -> 1.22.5
+- Google Chrome 95 -> 96
+- ChromeDriver 95 -> 96
+- Erlang 23.3.4.8 -> 23.3.4.9, 24.1.3 -> 24.1.5 (default)
```
Comment (bogyo210) [inline docs/reference/semaphore-changelog.md:25]: This should be on separate lines

Comment (d-stefanovic) [reply docs/reference/semaphore-changelog.md:25]: I will fix it for next changelog

**Pull request merged**